File: RSC-2019/Round-1/2/Solution.cpp
#include <iostream>

int main() {
    int n;
    std::cin >> n;
    while (n>0) {
        int t1, t2;
        std::cin >> t1 >> t2;
        if (t2 - t1 >= 5) {
            std::cout << t1;
            return 0;
        }
        n--;
    }
    std::cout << "doggy sad";
}

File: RSC-2019/Round-1/2/Solution.java
import java.util.Scanner;

class Solution {
    public static void main(String args[]) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        while (n>0) {
            int t1 = scanner.nextInt();
            int t2 = scanner.nextInt();
            if (t2 - t1 >= 5) {
                System.out.println(t1);
                System.exit(0);
            }
            n--;
        }
        System.out.println("doggy sad");
        scanner.close();
    }
}